src/CoreBundle/private/modules/MTTableManager/MTTableManager.class.php

```diff
@@ -441,17 +441,19 @@ public function getAutocompleteRecordList()
         $recordList = call_user_func(array($autoClassName.'List', 'GetList'), $this->getAutocompleteListQuery());
 
         $returnVal = [];
-        $returnVal[] = ['id' => ' ', 'text' => ' '];
+        $returnVal[] = ['id' => ' ', 'text' => ' ', 'html' => ' ', 'class' => 'd-none'];
```

Comment:
Better name it "css-class" (or similiar)?

At least here (in .php) it would be confusable with normal php classes.

Reply:
Changed to "cssClass" (without "-" so it is accessible in javascript as property)